lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -480,4 +480,13 @@ private function parseColumnCommentFromSQL(string $column, string $sql) : ?strin
 
         return '' === $comment ? null : $comment;
     }
+
+    private function removeDoctrineTypeFromDefault(?string $default) : ?string
+    {
+        if (preg_match('/^(.*)(--\(DC2Type:.*\))/', $default, $match)) {
```

Comment:
Maybe make the second group non-capturing?
```suggestion
        if (preg_match('/^(.*)(?:--\(DC2Type:.*\))/', $default, $match)) {
```